modules/update-release-images.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-release-images_{context}"]
+= Release images
+
+A release image is the delivery mechanism for a specific OCP version.
+It contains the release metadata, a CVO binary matching the release version, every manifest needed to deploy individual OpenShift Cluster Operators, and a list of SHA digest-versioned references to all container images that make up this OpenShift version.
+// to-do: check that "images", the final word of the sentence above, refers to Cluster Operator images.
```

Comment:
It's both cluster operator images and operand and other images used by those cluster operators.  For example, `cli-artifacts` is served by the console operator for anyone who wants to download an `oc` client from their cluster, but not used otherwise in the cluster, and "hosts a download service for some binaries" is sort of tiptoeing on the edge of what it means to be a console operand.

```console
$ oc adm release info --pullspecs quay.io/openshift-release-dev/ocp-release:4.13.0-x86_64
...
Images:
  NAME                                           PULL SPEC
  agent-installer-api-server                     quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:1d73cf64abffab71dc2e995c22158fcffdd2484131290cbc3fd520c4037d3929
  agent-installer-csr-approver                   quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:ce3e2e4aac617077ac98b82d9849659595d85cd31f17b3213da37bc5802b78e1
  agent-installer-node-agent                     quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:70397ac41dffaa5f3333c00ac0c431eff7debad9177457a038b6e8c77dc4501a
  ...
  baremetal-runtimecfg                           quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:ce9d408356abfc5fbb5b5bcb69a4eb8d49c47588747935a94ec8c31dfabe8eac
  cli                                            quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:800f0bb464dc9d622c3a670e503bee267670395c9bea0fb6247737b6f826ba7d
  cli-artifacts                                  quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:a196f1f9519404e034eb33f02667cccaede047edfaebc8e0081b07be73ce5f8d
  cloud-credential-operator                      quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:bcb76631bf77fe40297b07cc5e8f27753535e756fe6222a66697559cabdbfea2
  ...
  vsphere-csi-driver-syncer                      quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:9fd1196117528fe2226542484c902ca43d4d12fab37fab1d82711285ceba08e9
  vsphere-problem-detector                       quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:acca5e030186dbc1383e05db41091daffce400be2c44cd724eb0f235c22a8432
```